### ISSUE TITLE
Hardcoded deployment name in created service

### ISSUE THREAD

**Issue by blacksails** [jetstack/kube-lego#250]
It is a problem that the deployment name is hard coded in the selector created service. I want to run two instances of kube-lego in parallel, one using the staging url and the other using production.

https://github.com/jetstack/kube-lego/blob/6b5cb1375c44d7a98feea9eef96b8dda4a8fd65d/pkg/service/service.go#L106

**Comment by ankon** [comment]
(This is a duplicate of https://github.com/jetstack/kube-lego/issues/25)
